### ISSUE TITLE
Node.js Foundation Core Technical Committee (CTC) Meeting 2017-04-12

### ISSUE THREAD

**Issue by Trott** [nodejs/CTC#98]
## Time

**UTC Wed 12-Apr-2017 05:00 (05:00 AM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 11-Apr-2017 22:00 (10:00 PM) |
| US / Mountain | Tue 11-Apr-2017 23:00 (11:00 PM) |
| US / Central  | Wed 12-Apr-2017 00:00 (12:00 AM) |
| US / Eastern  | Wed 12-Apr-2017 01:00 (01:00 AM) |
| Amsterdam     | Wed 12-Apr-2017 07:00 (07:00 AM) |
| Moscow        | Wed 12-Apr-2017 08:00 (08:00 AM) |
| Chennai       | Wed 12-Apr-2017 10:30 (10:30 AM) |
| Tokyo         | Wed 12-Apr-2017 14:00 (02:00 PM) |
| Sydney        | Wed 12-Apr-2017 15:00 (03:00 PM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Core%20Technical%20Committee%20(CTC)+Meeting+2017-04-12&iso=20170412T05
* or http://www.wolframalpha.com/input/?i=5am+UTC%2C+Apr+12%2C+2017+in+local+time

## Links

* **Minutes Google Doc**: <https://docs.google.com/document/d/1fOmVjMRtX3mClvEnlA7tCpi0ihBWls_D6MvmWEKEDcs>
* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1pJoUGqOfNt4cVjLyPxJzfw3M509R7L-IQfXkHUDrp0Q>_

## Agenda

Extracted from **ctc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

## Invited

* Anna Henningsen @addaleax (CTC)
* Bradley Meck @bmeck (observer/GoDaddy/TC39)
* Ben Noordhuis @bnoordhuis (CTC)
* Сковорода Никита Андреевич @ChALkeR (CTC)
* Colin Ihrig @cjihrig (CTC)
* Evan Lucas @evanlucas (CTC)
* Franziska Hinkelmann @fhinkel (CTC)
* Jeremiah Senkpiel @Fishrock123 (CTC)
* Tracy Hinds @hackygolucky (observer/Node.js Foundation)
* Fedor Indutny @indutny (CTC)
* James M Snell @jasnell (CTC)
* Josh Gavant @joshgav (observer/Microsoft)
* Michael Dawson @mhdawson (CTC)
* Julien Gilli @misterdjules (CTC)
* Mikeal Rogers @mikeal (observer/Node.js Foundation)
* Brian White @mscdex (CTC)
* Myles Borins @MylesBorins (CTC)
* Ali Ijaz Sheikh @ofrobots (CTC)
* Jenn Turner @renrutnnej (observer/Node.js Foundation)
* Rod Vagg @rvagg (CTC)
* Seth Thompson @s3ththompson (observer/Google)
* Shigeki Ohtsu @shigeki (CTC)
* Steven R Loomis @srl295 (observer/IBM/ICU)
* Michaël Zasso @targos (CTC)
* Sakthipriyan Vairamani @thefourtheye (CTC)
* Trevor Norris @trevnorris (CTC)
* Rich Trott @Trott (CTC)

## Notes

The agenda comes from issues labelled with `ctc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Uberconference; participants should have the link & numbers, contact me if you don't.

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the CTC that's not worth putting on as a separate agenda item, this is a good place for it.

**Comment by Trott** [comment]
No issues in the agenda as of right now, which is a little surprising given the imminent-ish 8.0.0 release. If you have something that should/must go on the agenda, leave a comment or else the meeting seems likely to get canceled.

**Comment by Trott** [comment]
14 issues labeled ctc-review. Check 'em out as usual but maybe also remove the label for ones where it's no longer relevant?

* [module: remove require('.') with NODE_PATH compat](https://github.com/nodejs/node/pull/3384)
* [http: don't inherit from Object.prototype](https://github.com/nodejs/node/pull/6102)
* [errors, process: port internal/process errors to internal/errors](https://github.com/nodejs/node/pull/11294)
* [errors: assign error code to bootstrap_node created error](https://github.com/nodejs/node/pull/11298)
* [errors, child_process: migrate to using internal/errors](https://github.com/nodejs/node/pull/11300)
* [doc: add active, enroll and unenroll](https://github.com/nodejs/node/pull/11736)
* [async_hooks initial implementation](https://github.com/nodejs/node/pull/11883)
* [stream: move prefixed files into internal/streams.](https://github.com/nodejs/node/pull/11957)
* [src, buffer: add --pending-deprecation flag](https://github.com/nodejs/node/pull/11968)
* [lib,src: exit process on unhandled promise rejection cleanup](https://github.com/nodejs/node/pull/12010)
* [fs: Optimizations 3: The Revenge](https://github.com/nodejs/node/pull/12105)
* [Emeritus status for CTC](https://github.com/nodejs/CTC/issues/92)
* [Who should have access to the security repo ?](https://github.com/nodejs/CTC/issues/96)
* [Why do people need/want to be org owners](https://github.com/nodejs/CTC/issues/97)

**Comment by joshgav** [comment]
I'll be on east coast of US so won't be able to make the time this week (1am).

**Comment by Trott** [comment]
About 24 hours until the meeting and no agenda items. Anyone have anything they need to discuss (public or private)? Should we cancel?

**Comment by jasnell** [comment]
I'm likely to miss the call this week.

**Comment by cjihrig** [comment]
I'm going to miss the 1am meeting.

**Comment by mhdawson** [comment]
I'm not going to be able to make it. Too late in my timezone.

**Comment by targos** [comment]
So, are we cancelling?

**Comment by Trott** [comment]
> So, are we canceling?

@targos At this point, I'd say "yes, canceling". No agenda items are expected, and people in Europe have to figure out right about now if they want to get up early in the morning for this or not. So taking the best guess I can with the information we have: No meeting.

**Comment by Trott** [comment]
Sure seems canceled to me at this point. Closing. Feel free to re-open or comment if you think we should have the meeting after all...
